### DISCUSSION TITLE
PDF package

### DISCUSSION THREAD

**Discussion by SebGue** [Eomys/pyleecan#509] [Q&A]
Hello @BonneelP ,

do you already use some PFD creation package in Manatee? In case you don't I would give FPDF2 a try.
I also have ReportLab in use, but FPDF seems to be more straight forward and easier to use.

Best Regards, Sebastian

**Comment by BonneelP** [comment]
Hello,
For now we haven't worked on PDF creation so any package (with a proper licence) is fine for us :)

Best regards,
Pierre
